api/osb/check_visibility_plugin.go

```diff
@@ -114,23 +113,25 @@ func (p *checkVisibilityPlugin) checkVisibility(req *web.Request, next web.Handl
 				}
 			}
 		}
+		log.C(ctx).Errorf("Service plan %v is not visible on platform %v", planID, platform.ID)
```

Comment:
May be log also the plan name, as it will be easier to find out is this the plan you are looking for. The same for the other log message

Reply:
Now I saw, that we don't have the plan name here...